src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private final OperationType operationType;
+    private final String fruitName;
+    private final int quantity;
+
+    public FruitTransaction(OperationType operationType, String fruitType, int quantity) {
+        this.operationType = operationType;
+        this.fruitName = fruitType;
+        this.quantity = quantity;
+    }
+
+    public OperationType getOperationType() {
+        return operationType;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum OperationType {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private final String code;
+
+        OperationType(String code) {
+            this.code = code;
+        }
+
+        public String getCode() {
+            return code;
+        }

```

Comment:
It looks like you are duplicating the same enum here and in a separate OperationType class.